### DISCUSSION TITLE
求助大佬 打包一个软件呢

### DISCUSSION THREAD

**Discussion by hlf2016** [tw93/Pake#254] [Q&A]
https://httpie.io/

谢谢

**Comment by tw93** [comment]
https://github.com/tw93/Pake/wiki/GitHub-Actions-%E5%9C%A8%E7%BA%BF%E7%BC%96%E8%AF%91%E5%A4%9A%E7%B3%BB%E7%BB%9F%E7%89%88%E6%9C%AC

参考这个小白教程试试，有问题可以直接提出
